nexusutils/dataconverter/readers/rii_database/dispersion_functions.py

```diff
@@ -134,54 +146,55 @@ def sellmeier_polynomial(model_input: ModelInput):
     See https://refractiveindex.info/database/doc/Dispersion%20formulas.pdf
     for a reference.
     """
-    model_input.add_model_name(
-        model_input.dispersion_path,
-        'Sellmeier'
-    )
+    model_input.add_model_name(model_input.dispersion_path, "Sellmeier")
 
     a_units = []
     # Check for valid parameters
     for exp1 in model_input.coeffs[2:7:4]:
```

Comment:
Is this `exp1` also an obvious name for these functions?

Reply:
This stands for exponent 1, which is denoted as `e1` in the formula string. I can also write it out as `exponent1`.